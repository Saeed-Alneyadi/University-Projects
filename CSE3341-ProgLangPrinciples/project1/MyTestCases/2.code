procedure METHOD begin
    int x := 2023;

    output x;

    int y := x * x;

    output y ;

    int z := x + y;

    result;
end
