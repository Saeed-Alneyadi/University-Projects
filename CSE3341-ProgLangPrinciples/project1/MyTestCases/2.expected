PROCEDURE
ID[METHOD]
BEGIN
ID[int]
ID[x]
ASSIGN
COSNT[2023]
SEMICOLON
ID[output]
ID[x]
SEMICOLON
ID[int]
ID[y]
ASSIGN
ID[x]
MULTIPLY
ID[x]
SEMICOLON
ID[output]
ID[y]
SEMICOLON
ID[int]
ID[z]
ASSIGN
ID[x]
ADD
ID[y]
SEMICOLON
ID[result]
SEMICOLON
END
